docs/boxplot/BoxPlot.demo.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  AnnotationBoxPlotProps,
+  Plot,
+  Annotations,
+  Annotation,
+  SeriesPoint,
+} from 'react-plot';
+import { ScatterSeries, Axis } from 'react-plot';
+
+import random from 'random/dist/cjs';
+
+import { median, quartiles, max, min } from 'ml-stat/array';
+
+export default function BoxPlotExample() {
+  // dynamically generate some normally distributed data about the arbritary value of 300
+  const generator = random.normal(300, 100);
+  const jitter = random.normal(0, 0.1);
```

Comment:
Why you didn't use createRandomArray with a 'normal' distribution and a 'seed' from ml-spectra-processing as suggested in the issue ?

If something was missing you can always make a PR, it is one of our projects.

Reply:
I was focussing on the Telegram, sorry- will try what you've hinted.